grammars/fstar.cson

```diff
@@ -292,4 +304,4 @@ repository:
         match: '\\bop(?:_(?:Multiply|Star|Slash|Percent|Plus|Substraction|Equals|Less|Greater|Bang|Dollar|Amp|Dot|Hat|Colon|Pipe|Question))+\\b'
         name: 'entity.name.tag.fstar'
       }
-    ]
+    ]
```

Comment:
Your editor has stripped the trailing newline from the end of the file. You might want to put it back. 😉

Reply:
You are correct. I misclicked when accepting your change :(